tests/test_jwt_utils_import_error.py

```diff
@@ -0,0 +1,11 @@
+from unittest.mock import patch
+
+import pytest
+
+
+def not_working_test() -> None:  # TODO: finish me! - lbz/jwt_utils.py:26 mocking hell
```

Comment:
Please use `@pytest.mark.skip` to skip tests that you don't want to run. It will be at least shown in the report 😉  Also, it would be good to write a more descriptive TODO, at the first look, I don't have a clue what you wanted to achieve here.